compiler/luci/pass/src/VerifyQuantizedNodeType.h

```diff
@@ -107,12 +105,17 @@ class VerifyQuantizedNodeTypeBase : public luci::CircleNodeVisitor<bool>,
   bool visit(const luci::CircleSplitVOut *node);
   bool visit(const luci::CircleSqrt *node);
   bool visit(const luci::CircleStridedSlice *node);
-  virtual bool visit(const luci::CircleTanh *node) = 0;
   bool visit(const luci::CircleTranspose *node);
   bool visit(const luci::CircleTransposeConv *node);
   bool visit(const luci::CircleUnpack *node);
   bool visit(const luci::CircleUnpackOut *node);
 
+  // NOTE below nodes has differnent implementation for Qtype/Btype and
+  //      implementations exist in VerifyQuantizedNodeU8Type, VerifyQuantizedNodeS16Type
+  // bool visit(const luci::CircleLogistic *node);
+  // bool visit(const luci::CircleSoftmax *node);
+  // bool visit(const luci::CircleTanh *node);
```

Comment:
I agree with this change, following comment is just FYI after seeing https://github.com/Samsung/ONE/pull/8333#issuecomment-1021810821 :)

---

At first, I used `virtual bool visit(...) = 0;` to prevent mistake such like "An operation is implemented in `VerifyQuantizedNodeU8Type` but not implemented in `VerifyQuantizedNodeS16Type`".
It was because of following code.

```cpp
  bool visit(const luci::CircleNode *) { return true; }
```

Since `VerifyQuantizedNodeTypeBase` will be used only as base class, designating some of operations as pure virtual looked okay for me :)

Reply:
> "An operation is implemented in VerifyQuantizedNodeU8Type but not implemented in VerifyQuantizedNodeS16Type".
It was because of following code.

You got a point. This may give some problem in the future. Let's fix it when this happens.